contracts/proxy/transparent/TransparentUpgradeableProxy.sol

```diff
@@ -15,12 +15,7 @@ import {ProxyAdmin} from "./ProxyAdmin.sol";
  * include them in the ABI so this interface must be used to interact with it.
  */
 interface ITransparentUpgradeableProxy is IERC1967 {
-    /**
-     * @dev Upgrade the implementation of the proxy to `newImplementation`, and subsequently execute the function call
-     * encoded in `data`.
-     *
-     * See {UUPSUpgradeable-upgradeToAndCall}
-     */
+    /// @dev See {UUPSUpgradeable-upgradeToAndCall}
```

Comment:
we mostly use this style of comments:

```suggestion
    /**
     * @dev See {UUPSUpgradeable-upgradeToAndCall}
     */
```
is there consensus or what is the criteria to choose one or the other?

Reply:
(see Access Control)

Reply:
resolved: `///` is preferred for comments that require one or two lines, otherwise use `/** ...`